### DISCUSSION TITLE
apexcharts and tabler bad color tooltip in dark and light theme

### DISCUSSION THREAD

**Discussion by zatrudnitel** [tabler/tabler#1591] [Q&A] [answered]
help me please, i get this result
![image](https://github.com/tabler/tabler/assets/29029547/e0a5e7e9-65fa-47c1-b3f4-8194a7f7895c)
digits and date is gray color in dark theme, and I want them to be white as in the example - https://preview.tabler.io/charts.html?theme=dark
how to do it?
Thanks

**Comment by dheineman** [comment, marked as answer]
You might be missing some ApexChart options but it is hard to tell without any example code.

Have a look at the way the preview page [initialises](https://github.com/tabler/tabler/blob/72f868b617e238db1e761bb0830d6145b5937bac/demo/charts.html#L953-L1033) the "Social referrals" chart on the page you linked as it provides a bunch of options to ApexCharts. I am guessing maybe the `tooltip -> theme` option ([docs](https://apexcharts.com/docs/options/tooltip/)) might be wat you are looking for but i have not tested it myself.
